build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks.csproj

```diff
@@ -62,8 +62,8 @@
     <Compile Include="Xamarin.Android.Tools.BootstrapTasks\JdkInfo.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.ApiCompat" Version="5.0.0-beta.19606.1" IncludeAssets="none" />
-    <PackageReference Include="Microsoft.DotNet.GenAPI" Version="5.0.0-beta.19606.1" />
+    <PackageReference Include="Microsoft.DotNet.ApiCompat" Version="5.0.0-beta.20078.1" IncludeAssets="none" />
+    <PackageReference Include="Microsoft.DotNet.GenAPI" Version="5.0.0-beta.20078.1" />
```

Comment:
Any specific reason why to bump the version?

Reply:
I talked to Santi the person responsible and he said they are making some perf improvements, so I decide to bump the version.